src/objects/list/list.scss

```diff
@@ -40,3 +43,17 @@
   margin-left: math.div(size.$spacing-list-inline-gap, 2);
   margin-right: math.div(size.$spacing-list-inline-gap, 2);
 }
+
+@for $i from 2 through 3 {
+  .o-list--#{$i}-column {
+    @include media-query.breakpoint-classes($from: s, $to: xl) {
+      columns: #{$i};
+      @include fluid.grid-gap(
+        breakpoint.$s,
+        breakpoint.$xl,
+        size.$spacing-gap-fluid-min,
+        size.$spacing-gap-fluid-max
+      );
```

Comment:
Is it intentional that we're setting `grid-gap` rather than `column-gap` here?

Reply:
No I missed this, thanks!